Assets/Scripts/Managers/MainManager.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+// Manager for the Main Menu
+public class MainManager : MonoBehaviour
+{
+
+    #region Methods
+
+    public void PlayGame()
+    {
+        // Get last saved scene then load scene
+        // loading castle scene for now
+        SceneLoader.LoadLevel(2);
```

Comment:
I would like to avoid numbers, maybe you can make a constant CASTLE_SCENE=2 or something

Reply:
I was thinking about that today, good idea! I will update.